src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/session/GetSessionsDoubleBookingTest.kt

```diff
@@ -0,0 +1,169 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.integration.session
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.springframework.test.web.reactive.server.WebTestClient.BodyContentSpec
+import org.springframework.test.web.reactive.server.WebTestClient.ResponseSpec
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.enums.SessionConflict
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.enums.VisitStatus
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.sessions.VisitSessionDto
+import uk.gov.justice.digital.hmpps.visitscheduler.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.visitscheduler.model.entity.session.SessionTemplate
+import java.time.LocalDate
+
+@DisplayName("Get /visit-sessions")
```

Comment:
This display name is being shared by 3 other test classes. Could we change them to be more unique?

E.g.

@DisplayName("Get /visit-sessions - Double Booking tests")